server.js

```diff
@@ -1,12 +1,27 @@
+var path = require('path');
 var artsyEigenWebAssociation = require('artsy-eigen-web-association');
+var verificationFileName = path.resolve(__dirname, 'apple-developer-web-association.txt');
+var options = { 'headers': { 'Content-Type': 'text/plain' } };
+
 var express = require('express');
 var http = require('http');
 var morgan = require('morgan');
 var app = express();
 app.use(morgan('combined'));
 app.use('/(.well-known/)?apple-app-site-association', artsyEigenWebAssociation);
+
+app.get('/.well-known/apple-developer-domain-association.txt', function (req, res) {
+  res.sendFile(verificationFileName, options, function (error) {
+    if (error) {
+      console.log(error);
+      code = error.status >= 100 && error.status < 600 ? error.status : 500
+      response.status(code).end();
+    }
+  })
+})
```

Comment:
Might we need a similar route on www.artsy.net? If so, this could move into `artsy-eigen-web-association` which is shared between Force and this repo. (If not, this is fine.)

Reply:
I didn't add this to artsy-eigen-web-association because this route should only be necessary for the verification step and then can be removed. If we end up needing to do this verification again for whatever reason it probably makes sense to add there but if you'd rather it go in the library now I can make the change. Also I don't believe we need it in www.artsy.net for verification purposes just artsy.net.